src/Microsoft.AspNetCore.Mvc.Razor.Precompilation/Internal/RemoveStrongNameRewriter.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Cci;
+using Microsoft.Cci.MutableCodeModel;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Precompilation.Internal
+{
+    public class RemoveStrongNameRewriter : MetadataRewriter
+    {
+        public RemoveStrongNameRewriter(IMetadataHost host)
+            : base(host)
+        {
+        }
+
+        public override IAssembly Rewrite(IAssembly assembly)
+        {
+            var mutableAssembly = base.Rewrite(assembly) as Assembly;
```

Comment:
We might want to adopt the convention that I've seen in roslyn - use `CCI.Assembly` for these names. CCI tends to have a lot of overlap with type names in `System` or reflection